src/azul/pagination.py

```diff
@@ -0,0 +1,38 @@
+from azul.requests import parse_url_qs
+
+obsolete_pagination_parameters = ('search_after', 'search_after_uid', 'search_before', 'search_before_uid')
+'''
+    These pagination parameters are used internally, but should removed from the response that a client receives.
+'''
+
+
+def extract_pagination_params_from_url(url: str):
+    """
+       Helper method that extracts parameters for pagination from a url
+    """
+    query_params = parse_url_qs(url)
+    found_params = dict()
+    for x in obsolete_pagination_parameters:
+        if x in query_params:
+            found_params[x] = query_params[x]
+        else:
+            found_params[x] = None
+    return found_params
+
+
+def extract_pagination_params_from_response(json_response):
```

Comment:
I thought it would be strange to modify the response that was coming from the API. Hence the use of 2 data structures. 
I think renaming the function would be `ok`, how about something like `extract_internal_pagination_params_from_response`..... too wordy?

Reply:
A little wordy but acceptable. Alternatively I think `extract_internal_pagination_params(json_response)` works just as well.